### DISCUSSION TITLE
GLASS LAI Assimilation in LIS

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1495] [Q&A] [answered]
Hello, I wanted to know whether the latest version of NASA LIS 7.4 supports the GLASS LAI Assimilation? 
I was searching for that in the LIS user guide for version 7.4: https://lis.gsfc.nasa.gov/sites/default/files/LIS/public_7_4_releases/LIS_users_guide.22-jun-2022.pdf ;  but did not find any specifications for GLASS LAI in the Data Assimilation section. Although there is a paper: https://journals.ametsoc.org/view/journals/hydr/20/7/jhm-d-18-0237_1.xml 
which used NASA LIS for assimilating GLASS LAI. If this feature is enabled please share the specifications required for Data Assimilation in the LIS config file.

**Comment by dmocko** [comment, marked as answer]
Hi @Arijit22IITDelhi 

If you can, please upgrade to the public-7.5.0 release:
https://github.com/NASA-LIS/LISF/releases/tag/v7.5.0-public

The documentation for GLASS LAI is not in the Users' Guide.  We will need to fix that.

You can use these specifications:

```
Data assimilation set:                    "GLASS LAI"

Forcing perturbation algorithm:           "GMAO scheme"
Forcing perturbation frequency:           "1hr"
Forcing attributes file:                  ./forcing_attribs.txt
Forcing perturbation attributes file:     ./forcing_pert_attribs.txt

State perturbation algorithm:             "GMAO scheme"
State perturbation frequency:             "3hr"
State attributes file:                    ./noah_lai_attribs.txt
State perturbation attributes file:       ./noah_lai_pertattribs.txt

Observation perturbation algorithm:       "GMAO scheme"
Observation perturbation frequency:       "1da"
Observation attributes file:              ./GLASSLAI_attribs.txt
Observation perturbation attributes file: ./GLASSLAI_pertattribs.txt

GLASS LAI data directory:                 ./GLASS_LAI_data/
GLASS LAI data source:                    #   "AVHRR" or "MODIS"
GLASS LAI apply temporal smoother between 8-day intervals:   #  "1" for yes; "0" for no
```

The perturbation information is given in the journal article.

**Reply by Arijit22IITDelhi** [reply]
Thanks @dmocko. So with the above specifications, GLASS LAI Data Assimilation won't run in LIS 7.4?

**Reply by dmocko** [reply]
It will run in LIS-7.4.  We just ask for users to upgrade to the (edit: **latest**) official public release (7.5.0) if possible.

**Reply by Arijit22IITDelhi** [reply]
Thank you so much @dmocko
